src/app/js/admin/AdminContext.js

```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+const AdminContext = createContext();
+
+export const AdminContextProvider = ({ children }) => {
+    const [showEnrichmentColumns, setShowEnrichmentColumn] = React.useState(
```

Comment:
I think we directly import and call useState everywhere else

Reply:
You are right, I let too many control to my autocomplete...